package/firmware/linux-firmware/intel.mk

```diff
@@ -9,6 +9,16 @@ define Package/ibt-firmware/install
 endef
 $(eval $(call BuildPackage,ibt-firmware))
 
+Package/i915-firmware = $(call Package/firmware-default,Intel I915 Video Driver firmware)
+define Package/i915-firmware/install
+	$(INSTALL_DIR) $(1)/lib/firmware/i915
+	$(CP) \
+		$(PKG_BUILD_DIR)/i915/*.bin \
```

Comment:
The firmware files were already packaged in b6ac37110f19d130f608707b0bfcf8c1df60fbdf

Reply:
I have just noticed this, it is great. This change allows for finer adjustments and is better suited for scenarios where storage space is limited.

In my use case, for example NAS, the storage is not an issue, and would like to support as many CPUs as possible, just like Ubuntu, so it is better to deploy all the available i915 as a whole.